labs/lab-2-javascript-game/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title></title>
```

Comment:
Watch your indentation, here! 

You can use that same editor autoindent trick (select all, Command+Shift+P, autoindent, enter) on HTML, too!